src/policy/fees.cpp

```diff
@@ -1043,5 +1043,5 @@ CAmount FeeFilterRounder::round(CAmount currentMinFee)
     if ((it != feeset.begin() && insecure_rand.rand32() % 3 != 0) || it == feeset.end()) {
         it--;
     }
-    return *it;
+    return static_cast<CAmount>(*it);
```

Comment:
In commit "Make float <-> int casts explicit outside of test, qt, CFeeRate"

Nit: for primitive types we usually use C-style casts for brevity (in many places in this commit).

Reply:
Heh, I was assuming based on #10498 that we want to migrate. I dont care either way.

Reply:
Note that #10498 is only about non-primitive types.

Reply:
Ah, heh, well CAmount we stradle the line...cause its a primitive typedef we often pass it by reference. I'll leave it for now since its a bit more clear what going on.